usaspending_api/api_contracts/contracts/downloadCenter/DownloadCenter.md

```diff
@@ -103,4 +103,4 @@ This endpoint returns data corresponding to the latest data dictionary csv file.
 + `total_rows`: 393 (required, number)
 + `total_columns`: 12 (required, number)
 + `total_size`: `119.32KB` (required, string)
-+ `download_location`: `http://files-nonprod.usaspending.gov/docs/DATA+Transparency+Crosswalk.xlsx` (required, string)
++ `download_location`: `http://files.usaspending.gov/docs/DATA+Transparency+Crosswalk.xlsx` (required, string)
```

Comment:
Should we be concerned at all about this changing? Personally I don't see an issue with just not using the nonprod version of the key, but this might cause automated tests to fail if the crosswalk changes.